### PR TITLE
Added images pages support

### PR DESCRIPTION
**Pull request by thardwardy**
Added '/imghp' and 'images.google' to affected pages.
Replaced vector logo on images search page with image.

### CODE REVIEW

File: replace.js [hunk 1]
```diff
@@ -19,9 +21,11 @@ if(page == "/" || page == "/webhp") {
 	if(headerbg != null) {
 		headerbg.remove();
 	}
-
+	
+	// Replace svg logo used on the image search page.
+	document.getElementsByTagName("svg")[0].outerHTML = '<img class="jfN4p" src="logo" style="background:none" alt="Google" data-atf="1" data-frt="0" width="92" height="30">';
```
Comment (toydotgame) [inline replace.js:26]: Might update this with a string var for the boilerplate ID and some simpler JS code to identify the HTML `src` and `alt` attributes in a more open way. Matching an exact string is a bit too boilerplate IMO, and I think those `width` and `height` values might be JS controlled (potentially).

Comment (toydotgame) [reply replace.js:26]: is `document.querySelectorAll("[alt='Google']")` and some integration of `document.getElementsByClassName("jfN4p")` into that (idk how yet) simple enough yet enough selectors to select only the logo SVG/PNG?

**Review by toydotgame — CHANGES_REQUESTED** [review]
doesnt work bozo L + ratio

**Comment by thardwardy** [comment]
worked on stuff together in person, big re-write, closing

**Pull request closed without merging**